sspl_test/run_qa_test.sh

```diff
@@ -152,6 +156,17 @@ restore_cfg_services()
         conf "$global_config" set "server_node>$machine_id>site_id=DC01"
         conf "$global_config" set "server_node>$machine_id>rack_id=RC01"
         conf "$global_config" set "server_node>$machine_id>cluster_id=CC01"
+        if [ "${CVG_NAME}" == "null" ]
+            then
+            if [ "${CVG_INFO}" == "null" ]
+                then
+                `conf $global_config delete "server_node>$machine_id>storage>cvg"`
```

Comment:
deleting cvg key itself? means it never existed and we had created for tests ?

Reply:
Just I have added this condition that If CVG information is not available in the config file the I am creating this for test and deleting this mocked info after the test is complteted.